src/verification/index.ts

```diff
@@ -47,7 +51,11 @@ export default class Verification {
       )
     }
 
-    if (!this.verificationPublicKey) {
+    if (!this.getVerificationPublicKey) {
```

Comment:
Same suggestion as `Crypto` class, can this be checked in the constructor?

Reply:
[ditto](https://github.com/opengovsg/formsg-javascript-sdk/pull/113#discussion_r1981050298)